clang/test/AST/Interp/intap.cpp

```diff
@@ -44,6 +44,24 @@ static_assert(MulA * MulB == 50, ""); // ref-error {{not an integral constant ex
 static_assert(MulA * 5 == 25, "");
 static_assert(-1 * MulB == -7, "");
 
+
+constexpr _BitInt(4) DivA = 2;
+constexpr _BitInt(2) DivB = 1;
+static_assert(DivA / DivB == 2, "");
+
+constexpr _BitInt(4) DivC = DivA / 0; // ref-error {{must be initialized by a constant expression}} \
```

Comment:
I'd appreciate another test along the lines of:
```
constexpr int bottom = -1;
constexpr int top = INT_MIN;
constexpr int nope = top / bottom;
constexpr int noooo = top % bottom;
```

Reply:
Did you really mean `int`? Or `_BitInt`?

Reply:
I meant `int`; basically, it's catching the other form of UB that's not division by zero.

Reply:
Using `int` means it won't use `IntegralAP` at all though. I've added those tests with `_BitInt(32)`.

Reply:
Ah, thank you!